nistats/datasets.py

```diff
@@ -237,8 +237,68 @@ def _glob_spm_multimodal_fmri_data():
             _uncompress_file(archive_path)
         except:
             print("Archive corrupted, trying to download it again.")
-            return fetch_spm_multimodal_fmri_data(data_dir=data_dir,
-                                                  data_name="",
-                                                  subject_id=subject_id)
+            return fetch_spm_multimodal_fmri(data_dir=data_dir,
+                                             data_name="",
+                                             subject_id=subject_id)
 
     return _glob_spm_multimodal_fmri_data()
+
+
+def fetch_fiac_first_level(data_dir=None, verbose=1):
+    """ Download a first-level fiac fMRI dataset (2 sessions)
+
+    Parameters
+    ----------
+    data_dir: string
+        directory where data should be downloaded and unpacked.
+    """
+    data_dir = _get_dataset_dir('', data_dir=data_dir, verbose=verbose)
+
+    def _glob_fiac_data():
+        """glob data from subject_dir."""
+        _subject_data = {}
+        subject_dir = os.path.join(data_dir, 'nipy-data-0.2/data/fiac/fiac0')
+        for session in [1, 2]:
+            # glob func data for session s + 1
+            session_func = os.path.join(subject_dir, 'run%i.nii.gz' % session)
+            if not os.path.isfile(session_func):
+                print('Missing functional scan for session %i.' % session)
+                return None
+            else:
```

Comment:
you don't really need the `else` and can dedent the following line one level since you have an early return in the if clause. You have a few similar patterns below.